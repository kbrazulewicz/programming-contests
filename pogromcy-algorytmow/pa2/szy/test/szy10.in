10
40
8940293
1793275
11705592
10211989
27343754
241277
34197024
8366443
27988132
36393701
7636195
5106444
39797666
12142102
9670643
24212473
33335352
32435454
17697994
39738974
3840869
34313502
17384989
17619799
39569438
26527438
36524121
23103903
30779120
36256760
17274134
39184689
8894644
3743155
35122179
11587862
27909880
9716290
23593371
1662911
339024233