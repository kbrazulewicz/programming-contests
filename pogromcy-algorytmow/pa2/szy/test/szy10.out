1011101000111000000000111100001001110011
